devel/ci-cluster.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will build an entirely new testing environment using kind.
+# This is intended to be run in a CI environment and *not* for development.
+# It is not optimised for quick, iterative development.
+
+export_logs() {
+  echo "Exporting cluster logs to artifacts..."
+  "${SCRIPT_ROOT}/cluster/export-logs.sh"
+}
+
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+export REPO_ROOT="${SCRIPT_ROOT}/.."
+source "${SCRIPT_ROOT}/lib/lib.sh"
+
+# Configure PATH to use bazel provided e2e tools
+setup_tools
+
+export SERVICE_IP_PREFIX="10.0.0"
```

Comment:
comment: this is defaulted in lib.sh: https://github.com/jetstack/cert-manager/blob/4dfb5add863ce969473c0eef38218d26e841eef9/devel/lib/lib.sh#L31

I don't think this needs to be changed though given this file is just moving code directly from `devel/ci-run-e2e.sh`